app/start/global.js

```diff
@@ -1,5 +1,7 @@
 
 var App = require('positron');
+App.command = require('../console/commands/Command');
```

Comment:
No need for this when you load command service with the help of service provider as I mentioned above.